src/ui/Label/index.scss

```diff
@@ -1,10 +1,5 @@
 @import '../../styles/variables';
 
-.sendbird-label {
-  -webkit-user-select: none;
-  -webkit-touch-callout: none;
-}
```

Comment:
Hey, thanks for this, but this is a workaround for certain mobile devices... anyways, will scope it down so that it works on only those devices ~

Reply:
Also, sadly, we are not allowed to merge these PRs from ext. I will make a  separate PR and add you as courtesy

Reply:
https://github.com/sendbird/sendbird-uikit-react/pull/625